pkg/loop/internal/pb/keystore/keystore.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/keystore;keystorepb";
+
+package loop.internal.pb.keystore;
+
+service Keystore {
+  rpc Sign(SignRequest) returns (SignResponse);
+  rpc SignBatch(SignBatchRequest)returns (SignBatchResponse);
+  rpc Verify(VerifyRequest)returns (VerifyResponse);
+  rpc VerifyBatch(VerifyBatchRequest)returns (VerifyBatchResponse);
+
+  rpc List(ListRequest)returns (ListResponse);
+
+  rpc RunUDF(RunUDFRequest)returns (RunUDFResponse);
+
+  //
+  rpc Import(ImportRequest)returns(ImportResponse);
+  rpc Export(ExportRequest)returns(ExportResponse);
+
+  rpc Create(CreateRequest)returns(CreateResponse);
+  rpc Delete(DeleteRequest)returns(DeleteResponse);
+
+  rpc AddTag(AddTagRequest)returns(AddTagResponse);
+  rpc RemoveTag(RemoveTagRequest)returns(RemoveTagResponse);
+  rpc ListTags(ListTagsRequest)returns(ListTagsResponse);
+}
+
+message SignRequest {
+  bytes keyID = 1;
+  bytes data = 2;
+}
+
+message SignResponse {
+  bytes data = 1;
+}
+
+message SignBatchRequest {
+  bytes keyID = 1;
+  repeated bytes data = 2;
+}
+
+message SignBatchResponse {
+  repeated bytes data = 1;
+}
+
+message VerifyRequest {
+  bytes keyID = 1;
+  bytes data = 2;
+}
+
+message VerifyResponse {
+  bool valid = 1;
+}
+
+message VerifyBatchRequest {
+  bytes keyID = 1;
+  repeated bytes data = 2;
+}
+
+message VerifyBatchResponse {
+  repeated bool valid = 1;
+}
+
+message ListRequest {
+  repeated string tags = 1;
+}
+
+message ListResponse {
+  repeated bytes keyIDs = 1;
+}
+
+message RunUDFRequest {
+  string udfName = 1;
```

Comment:
Is the `udf` prefix adding value here?
```suggestion
  string name = 1;
```
Or would it be more clear as `id`?
```suggestion
  string id = 1;
```

Reply:
Renamed to `name` in [`9bc2713` (#837)](https://github.com/smartcontractkit/chainlink-common/pull/837/commits/9bc2713bc46d6518a758830828454e7077039dbe)